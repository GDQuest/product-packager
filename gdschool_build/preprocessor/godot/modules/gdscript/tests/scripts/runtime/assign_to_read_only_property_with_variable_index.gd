func test():
	var state = PhysicsDirectBodyState3DExtension.new()
	var prop = &"center_of_mass"
	assign(state, prop)
	state.free()

func assign(state, prop):
	state[prop].x = 1.0
