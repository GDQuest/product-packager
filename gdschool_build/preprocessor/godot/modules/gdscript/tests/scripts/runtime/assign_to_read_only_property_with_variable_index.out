GDTEST_RUNTIME_ERROR
>> SCRIPT ERROR
>> on function: assign()
>> runtime/assign_to_read_only_property_with_variable_index.gd
>> 8
>> Cannot set value into property "center_of_mass" (on base "PhysicsDirectBodyState3DExtension") because it is read-only.
